docs/OpenTelemetry.md

```diff
@@ -0,0 +1,44 @@
+| `fga-client.request.store_id`  | `string` | The store ID that was sent as part of the request                                                                       |
+| `fga-client.request.model_id`  | `string` | The authorization model ID that was sent as part of the request, if any                                                 |
+| `fga-client.request.client_id` | `string` | The client ID associated with the request, if any                                                                       |
+| `fga-client.user`              | `string` | The user that is associated with the action of the request for check and list objects                                   |
+| `fga-client.request.retries`   | `int`    | The number of retries attempted (starting from 1 for the original request). Deprecated, use `http.request.resend_count` |
+| `http.request.resend_count`    | `int`    | The number of retries attempted (starting from 1 for the original request)                                              |
+| `http.status_code`             | `int`    | The status code of the response. Deprecated, use `http.response.status_code`                                            |
+| `http.response.status_code`    | `int`    | The status code of the response                                                                                         |
+| `http.method`                  | `string` | The HTTP method for the request. Deprecated, use `http.request.method`                                                  |
+| `http.request.method`          | `string` | The HTTP method for the request                                                                                         |
+| `http.host`                    | `string` | Host identifier of the origin the request was sent to                                                                   |
+| `url.scheme`                   | `string` | HTTP Scheme of the request (`http`/`https`)                                                                             |
+| `url.full`                     | `string` | Full URL of the request                                                                                                 |
+| `user_agent.original`          | `string` | User Agent used in the query                                                                                            |
+| `http.client.request.duration` | `int`    | The total request time for FGA requests                                                                                 |
+| `http.server.request.duration` | `int`    | The amount of time the FGA server took to internally process nd evaluate the request                                    |
+
+## Example
+
+There is an [example project](https://github.com/openfga/java-sdk/blob/main/example/opentelemetry) that provides some guidance on how to configure OpenTelemetry available in the examples directory.
```

Comment:
Do we plan to add an example prior to release or shall we remove this?

Reply:
OK to leave in, I have an example lined up for the release 👍

Reply:
Can we add the example as part of this PR?